tests/python/rest_api/utils.py

```diff
@@ -110,13 +126,17 @@ def export_v2(
         return None
 
     # check status of background process
-    return wait_and_download_v2(
+    result = wait_and_download_v2(
         endpoint.api_client,
         rq_id,
         max_retries=max_retries,
         interval=interval,
         download_result=download_result,
     )
+    if not download_result and return_request_id:
```

Comment:
`return_request_id` seems redundant. If we always return the result when `download_result` is true, why not return the request ID when it's false?